apps/wizard/pages/chart_diff/app.py

```diff
@@ -22,8 +27,12 @@
 # TODO: simplify this
 SOURCE_ENV = DB_HOST  # "staging-site-streamlit-chart-approval"
 SOURCE_API = f"https://api-staging.owid.io/{SOURCE_ENV}/v1/indicators/"
-# TODO: switch to production once we are ready
-TARGET_ENV = "staging-site-master"
+
+if ENV_FILE_PROD.exists():
+    TARGET_ENV = ENV_FILE_PROD
+else:
+    log.warning(f"Production env file {ENV_FILE_PROD} not found, comparing against staging-site-master")
```

Comment:
I'd probably add a streamlit `st.warning` or something here, to signal this on the end user. It could have big implications if they don't know, right?

Reply:
Added. It doesn't write anything to DB and `.env.prod` is read-only, so those implications are not as bad, but it's still good to know.